### PR TITLE
feat: Create album screen

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [MOPLAT-281]

### Description

image

### PR Checklist

- [x] I tested my changes on **iOS** and **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-281]: https://artsyproduct.atlassian.net/browse/MOPLAT-281?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/palette/elements/Separator/ShadowSeparator.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components/native"
+import { Separator } from "./Separator"
+
+export const ShadowSeparator = styled(Separator)`
```
Comment (pvinis) [inline src/palette/elements/Separator/ShadowSeparator.tsx:4]: do we need this to be here? do we have other screens that need this?

**Review by patrinoua — APPROVED** [review]
Looks great!

**Comment by pvinis** [comment]
oh well, merged lol. i guess we are doing it on the next ticket, or at least a new PR.

File: src/palette/elements/Separator/ShadowSeparator.tsx (hunk 1 above)
Comment (rajsam003) [reply src/palette/elements/Separator/ShadowSeparator.tsx:4]: Its 

> oh well, merged lol. i guess we are doing it on the next ticket, or at least a new PR.

Yess

Comment (rajsam003) [reply src/palette/elements/Separator/ShadowSeparator.tsx:4]: There is no other screen atm that needs it. I added it here as it looks like a palette component. Let me know if you want me to remove it.

**Pull request merged**